packages/halo-nonce-pcd/src/CardBody.tsx

```diff
@@ -0,0 +1,26 @@
+import { FieldLabel, Separator, Spacer, TextContainer } from "@pcd/passport-ui";
+import styled from "styled-components";
+import { HaLoNoncePCD } from "./HaLoNoncePCD";
+
+export function HaLoNonceCardBody({ pcd }: { pcd: HaLoNoncePCD }) {
+  return (
+    <Container>
+      <p>Zuzalu Memento</p>
```

Comment:
what is this a memento for?

```
Zuzalu Memento
(crypto words) 0x3783278
(more crypto) 0x37938723
```

seems slightly rough

this is also a general challenge with PCD cards though---Ethereum PCD has the same issue (just says "ETHEREUM" and then a bare address)

recommend thinking about in future / maybe for the app-based passport

Reply:
also I don't think the CardBody needs to say Zuzalu Memento. won't the card title already say that?

Reply:
Yep this is a temporary one. It will be replaced with a stamp that has a specific background